application/models/SurveysGroupsettings.php

```diff
@@ -418,7 +418,12 @@ protected static function translateOptionLabels($instance, $attribute, $value)
         } elseif ($attribute == 'owner_id' && $value != -1) {
             $instance->oOptions->owner = "";
             $instance->oOptions->ownerLabel = "";
-            $oUser = User::model()->findByPk($instance->oOptions->{$attribute});
+            /* \User|false[] see mantis #19426 */
```

Comment:
Is this expected here?

Reply:
Lakh of @var

Reply:
`/* @var array $oStaticUsers \User or null see mantis #19426 */`